### DISCUSSION TITLE
Добавьте обход блокировки твиттера/x

### DISCUSSION THREAD

**Discussion by Dr-Green007** [dovecoteescapee/ByeDPIAndroid#131] [Ideas]
Пожалуйста добавьте обход этого социалки и тогда это станет моим самый лучшим обходом замедление Рунета на телефоне

**Comment by andruxin68** [comment]
Вчера работало. Сегодня почему-то перестал загружать

**Comment by Konserva228** [comment]
> Пожалуйста добавьте обход этого социалки и тогда это станет моим самый лучшим обходом замедление Рунета на телефоне

Полазь в теме про Ютуб. Там выкладывают настройки для разных операторов. Я сперва запускал это приложение с настройками по-умолчанию, и у меня открывался только rutracker.org Потом нашёл настройки для своего мобильного Мегафона и, вроде как, стало открываться вообще все, том числе и твиттер/Х, кроме xvideos.com (не знаю почему так. Создал тему с вопросом). Ютуб на телефоне тоже летать стал максимально

**Reply by Dr-Green007** [reply]
> > Пожалуйста добавьте обход этого социалки и тогда это станет моим самый лучшим обходом замедление Рунета на телефоне
> 
> Полазь в теме про Ютуб. Там выкладывают настройки для разных операторов. Я сперва запускал это приложение с настройками по-умолчанию, и у меня открывался только rutracker.org Потом нашёл настройки для своего мобильного Мегафона и, вроде как, стало открываться вообще все, том числе и твиттер/Х, кроме xvideos.com (не знаю почему так. Создал тему с вопросом). Ютуб на телефоне тоже летать стал максимально

Пробовал, ни один не работает, только рутрекер, инстаграм и ютюб быстрее стал. У меня провайдер Орион Телеком, если вы нашли подходящие настройки для этого провайдера, то скидывайте сюда или я разберусь потом и скину

**Reply by Konserva228** [reply]
То есть у тебя проблема когда ты пытаешься использовать телефон по вайфаю через роутер с интернетом от Ориона, да?

А через мобильный интернет как? Пробовал? Или тариф слишком неподходящий для таких экспериментов?

**Reply by Konserva228** [reply]
Я вот сейчас тоже экспериментирую. От dns некоторые сайты перестают заходить. С 1.1.1.1 заходит везде (при остальных настройках от мегафона), с 8.8.8.8 перестаёт заходить в инстаграм. Но я не понимаю с чем это связано, к сожалению - не специалист. Может операторы какие-то dns у себя блочат?

**Reply by Dr-Green007** [reply]
> То есть у тебя проблема когда ты пытаешься использовать телефон по вайфаю через роутер с интернетом от Ориона, да?
> 
> А через мобильный интернет как? Пробовал? Или тариф слишком неподходящий для таких экспериментов?

С мобильном связем всё также. Возможно ты прав, ведь орион по сути сибирский ростелеком, так что есть шансы что как то заблокировал доступ к твиттеру

**Reply by Konserva228** [reply]
Попробуй другие dns. Мало ли. Ещё раз говорю, что я не специалист, но аналогия с моей ситуацией подсказывает логически пробежаться хотя бы по вариантам dns от 1.1.1.1 до 9.9.9.9 Может ещё какие цифры попробуй. Телефон же от этого не взорвётся... Извини, если зря потрачу твоё время :)

**Reply by Dr-Green007** [reply]
> Попробуй другие dns. Мало ли. Ещё раз говорю, что я не специалист, но аналогия с моей ситуацией подсказывает логически пробежаться хотя бы по вариантам dns от 1.1.1.1 до 9.9.9.9 Может ещё какие цифры попробуй. Телефон же от этого не взорвётся... Извини, если зря потрачу твоё время :)

Ни то и ни то, я уже перебровалал настройки всё равно нет :/ но в любом случае я использую антизапрет чтобы заходить в твиттер, а этот бай для инстаграмма, рутрекера и конечно же ускорить ютюб

[a comment by an automated account is not shown]
